bertopic/_utils.py

```diff
@@ -147,3 +147,29 @@ def validate_distance_matrix(X, n_samples):
         raise ValueError("Distance matrix cannot contain negative values.")
 
     return X
+
+
+def get_unique_distances(dists: np.array, noise_min=1e-3, noise_max=1e-1) -> np.array:
```

Comment:
What are the reasons for choosing these noise values? The max noise seems rather high still. Wouldn't that affect the actual distances too much?

Have you checked whether the noise only separates the identical distances whilst keeping the order in check? For instance, let's assume we have the following distances:

```python
distances = [0, 0.15, 0.2, 0.2, 0.3]
```

With the noise as is currently implemented, can we be 100% percent sure that the identical values (i.e., `0.2`) remain lower than the next highest value (i.e., `0.3`) but higher than the previous lower value (i.e., `0.15`)?
```

Reply:
I was missing 0 at `1e-1`. 

1. I changed the defaults of the min.
2. Reimpleted the noise adding to make sure the ascending distance order is preserved.